### ISSUE TITLE
use main/module differentiation in package.json

### ISSUE THREAD

**Issue by jlowcs** [inorganik/countUp.js#262]
```
[ ] Bug
[x] Feature request

```

## Description

Could we have `main` from `package.json` point to the umd bundle, and add a `module` property that points to the ESM module?

From my understanding, that's how you differentiate a ESM bundle from the old non-ESM format in `package.json`.

```json
{
        "main": "./dist/countUp.umd.js",
        "module": "./dist/countUp.min.js"
}
```

Some examples:
https://github.com/microsoft/tslib/blob/main/package.json#L25
https://github.com/graphql/graphql-js/blob/main/package.json#L8
https://github.com/styled-components/styled-components/blob/main/packages/styled-components/package.json#L6

**Comment by jlowcs** [comment]
Related to https://github.com/inorganik/countUp.js/issues/218

**Comment by inorganik** [comment]
Cool, I didn't know about this! I'll make the change, thanks

**Comment by inorganik** [comment]
Ok, it's released in v2.0.8
